detox/e2e/test/channel_info/header.e2e.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
```

Comment:
Please add this file in the file header. The other files will have the same header as addressed in a separate PR so no need to add them there.

```suggestion

// *******************************************************************
// - [#] indicates a test step (e.g. # Go to a screen)
// - [*] indicates an assertion (e.g. * Check the title)
// - Use element testID when selecting an element. Create one if none.
// *******************************************************************

```